main_test.go

```diff
@@ -1,17 +1,41 @@
 package main
 
 import (
+	"fmt"
+	"net/url"
 	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
 	"testing"
 
+	"github.com/google/uuid"
+	"github.com/hashicorp/consul/sdk/freeport"
 	"github.com/rogpeppe/go-internal/testscript"
 	"github.com/stretchr/testify/require"
 
 	"github.com/smartcontractkit/chainlink/v2/core"
+	"github.com/smartcontractkit/chainlink/v2/core/config/env"
 	"github.com/smartcontractkit/chainlink/v2/core/static"
+	"github.com/smartcontractkit/chainlink/v2/internal/testdb"
 	"github.com/smartcontractkit/chainlink/v2/tools/txtar"
 )
 
+// special files can be included to allocate additional test resources
+const (
+	// testDBName triggers initializing of a test database.
+	// The URL will be set as the value of an env var named by the file.
+	//
+	//	-- testdb.txt --
+	//	CL_DATABASE_URL
+	testDBName = "testdb.txt"
```

Comment:
i don't see these files in the PR. what's happening? are tests dynamically creating them or something?

Reply:
They are embedded in the `*.txtar` files as described in the comment:
```
-- testdb.txt --
	CL_DATABASE_URL
```